scripts/idl/matter_idl_parser.py

```diff
@@ -182,18 +181,27 @@ def info_priority(self, _):
     def debug_priority(self, _):
         return EventPriority.DEBUG
 
+    def event_fabric_sensitive(self, _):
+        return EventQuality.FABRIC_SENSITIVE
+
+    def event_qualities(selt, qualities):
+        return set(qualities)
+
     def timed_command(self, _):
-        return CommandAttribute.TIMED_INVOKE
+        return CommandQuality.TIMED_INVOKE
 
-    def command_attributes(self, attrs):
+    def fabric_command(self, _):
```

Comment:
```suggestion
    def fabric_scoped_command(self, _):
```

Reply:
Updated (needed to change lark as well).